# goal
geometry_msgs/PoseStamped goal  #Goal location in world frame 
---
# result
bool status
---
# feedback

